### PR TITLE
Use PointerEvent interface to enable pointer capturing

### PR DESCRIPTION
**Pull request by beru**
By using the `setPointerCapture()` method of the Element interface, graph scrolling would be more comfortable. It seems using PointerEvent is necessary for  the method to work so several event handlings are also updated.

I still haven't tested if the change is compatible with touch-screen devices. 

https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture
https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent

### CODE REVIEW

**Comment by lutzroeder** [comment]
@beru can you add some context which scrolling scenario, operating system, browser and device this is targeting?

**Comment by beru** [comment]
@lutzroeder I use desktop app Netron, Version 6.8.2 on Windows 10. I scroll by clicking and dragging with a mouse.

Here's the scrolling scenario.

## Before

The scrolling is unexpectedly interrupted when a mouse cursor goes of a window during dragging.

before

## After

By using `setPointerCapture()`, the scrolling isn't interrupted when a mouse cursor goes of a window while dragging.

after

**Comment by lutzroeder** [comment]
@beru does not work in Safari on macOS.

File: source/view.js [hunk 1]
```diff
@@ -344,21 +344,19 @@ view.View = class {
...
+            this._events.pointerdown = (e) => this._pointerDownHandler(e);
         }
         const graph = this._element('graph');
         if (graph) {
             graph.focus();
         }
         graph.addEventListener('scroll', this._events.scroll);
         graph.addEventListener('wheel', this._events.wheel, { passive: false });
-        graph.addEventListener('mousedown', this._events.mousedown);
         if (this._host.environment('agent') === 'safari') {
```
Comment (beru) [inline source/view.js:356]: Due to this condition, Safari doesn't use pointerdown event.

The pointerdown event is supported from Safari 13.
https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerdown_event#browser_compatibility

Safari 13 was released on September 20, 2019
https://en.wikipedia.org/wiki/Safari_(web_browser)

**Comment by lutzroeder** [comment]
@beru `e.touches` does not exist on `PointerEvent`, throws on iPad. Can you validate the commit at TIP.

**Comment by beru** [comment]
@lutzroeder

> e.touches does not exist on PointerEvent, throws on iPad.

Oh I see... sorry for not testing on touch devices.

> Can you validate the commit at TIP.

I've tested b22093ccefdd5f97c161c8b7662225e7597bf669. It works well at my side.

**Pull request closed without merging**